peer_test.go

```diff
@@ -191,12 +191,12 @@ func TestPeerRemoveClosedConnection(t *testing.T) {
 
 func TestPeerConnectCancelled(t *testing.T) {
 	WithVerifiedServer(t, nil, func(ch *Channel, hostPort string) {
-		ctx, cancel := NewContext(109 * time.Millisecond)
+		ctx, cancel := NewContext(100 * time.Millisecond)
```

Comment:
That's what I'm assuming too, I'm pretty sure the test didn't need that extra 9ms!